{"name": "test_mulFloat[0.1-0.2-0.02]", "status": "failed", "statusDetails": {"message": "AssertionError: 实际结果0.020000000000000004与预期结果0.02不一致\nassert 0.020000000000000004 == 0.02", "trace": "self = <pytestSecond.test.testCalculator.TestCalculator object at 0x03BB0940>, a = 0.1, b = 0.2, expected = 0.02\n\n    @pytest.mark.run(order=3)\n    @pytest.mark.parametrize(['a', 'b', 'expected'], getData(\"mul\", \"floatDatas\"))\n    def test_mulFloat(self, a, b, expected):\n        result = self.cal.mul(a, b)\n>       assert result == expected, f'实际结果{result}与预期结果{expected}不一致'\nE       AssertionError: 实际结果0.020000000000000004与预期结果0.02不一致\nE       assert 0.020000000000000004 == 0.02\n\ntestCalculator.py:88: AssertionError"}, "parameters": [{"name": "a", "value": "0.1"}, {"name": "b", "value": "0.2"}, {"name": "expected", "value": "0.02"}], "start": 1606204073341, "stop": 1606204073342, "uuid": "6eb0358f-212f-4064-8403-731bd4aced0b", "historyId": "8f0910faa4fcc5074465f5a115bfb78f", "testCaseId": "68b3df60cbff2423ec14039eaec5edd9", "fullName": "testCalculator.TestCalculator#test_mulFloat", "labels": [{"name": "tag", "value": "run(order=3)"}, {"name": "suite", "value": "testCalculator"}, {"name": "subSuite", "value": "TestCalculator"}, {"name": "host", "value": "DESKTOP-103B3LA"}, {"name": "thread", "value": "47728-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testCalculator"}]}